static/.well-known/security.txt

```diff
@@ -1,11 +1,8 @@
 # Found a bug? Report it here:
 Contact: https://app.zerocopter.com/en/rd/f9afbf32-ce59-48ed-96f2-ac4410595aa4
-Contact: https://app.zerocopter.com/nl/rd/f9afbf32-ce59-48ed-96f2-ac4410595aa4
```

Comment:
Please don’t delete the non-english versions. Clarification with comment instead?

Reply:
ow, that are actually 4 different urls?
That is not clear at all!

Reply:
Added comments to point make it more distinct.